src/services/postNoteService.js

```diff
@@ -0,0 +1,6 @@
+import axios from "axios";
+
+const postNoteService = (newNote, header) =>
+	axios.post("/api/notes", { note: newNote }, { headers: header });
```

Comment:
you can keep this in try catch just to catch any error if it occues

Reply:
Yes, I have done the error handling in the place I called the function 👍🏽